### DISCUSSION TITLE
Sdkman-cli-native in production for gradle

### DISCUSSION THREAD

**Discussion by jcralbino** [sdkman/sdkman-cli-native#196] [Q&A] [answered]
Hello
I am planning in using sdkman to support our docs-as-code environment 

I would prefer to use this rust implementation, instead of the bash one.

Can you confirm if I would find any issues in using sdkman to install Java jdk requirements for asciidoctorJ, and gradle that will be  used for antora ( via gradle plugin)

**Comment by marc0der** [comment, marked as answer]
Hi @jcralbino,
If you install SDKMAN via the normal method on one of the supported platforms, you are already using the Rust binaries. For commands that haven't been ported yet, we automatically fall back to bash commands.

**Reply by jcralbino** [reply]
Hello
Thanks , good to know
But we still need to have then the mingwbb installed?

Regarding of the installations I mentioned of Java , gradle and asciidoctor that are supported, can you confirm if those were migrated ?

Thanks
